solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -2562,20 +2562,20 @@ public static Object skewed(Object likely, Object unlikely) {
    * A variant of {@link org.apache.solr.client.solrj.impl.CloudHttp2SolrClient.Builder} that will
    * randomize some internal settings.
    */
-  public static class CloudHttp2SolrClientBuilder extends CloudHttp2SolrClient.Builder {
+  public static class RandomizingCloudHttp2SolrClientBuilder extends CloudHttp2SolrClient.Builder {
```

Comment:
Name change makes sense.

Reply:
Since this name change didn't actually affect any other files - is this class even used???

Reply:
So, it is called from `SolrTestCaseJ4.getCloudHttp2SolrClient()`, who does `return new RandomizingCloudHttp2SolrClientBuilder(cluster).build();`

`getCloudHttp2SolrClient()` is only ever used by one method `CloudHttp2SolrClientBadInputTest.testDeleteByIdReportsInvalidIdLists`, who I don't actually think cares that it might have randomization.    

It appears to be a "good idea" that never went anywhere...    We could move it to that one class, or maybe look to use it in more places?   Dunno.